Indexers packages fails to built for main branch due to the absence of VERSION file

### Description

In [Nightly 27/02/2025](https://github.com/wazuh/wazuh-qa-automation/issues/1070) it has been detected that indexers packages are failing to construct:

**Action**: https://github.com/wazuh/wazuh-indexer/actions/runs/13555975035/job/37890237857

```
Run echo "version=$(<VERSION)" >> "$GITHUB_OUTPUT"

/home/runner/work/_temp/0cccb385-9ace-49b5-ae41-f68eb14bd1f3.sh: line 1: VERSION: No such file or directory

Error: Process completed with exit code 1.
```

The main branch no longer contains a VERSION file. Instead, it now has a VERSION.json file. This action should be adapted accordingly.

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -158,7 +158,7 @@ jobs:
 
       - name: Get version
         id: version
-        run: echo "version=$(bash build-scripts/product_version.sh)" >> "$GITHUB_OUTPUT"
+        run: echo "version=$(jq .version VERSION.json)" >> "$GITHUB_OUTPUT"
 
       - name: Build with Gradle
         working-directory: ./plugins/${{ matrix.plugins }}
@@ -201,7 +201,7 @@ jobs:
 
       - name: Get version
         id: version
-        run: echo "version=$(bash build-scripts/product_version.sh)" >> "$GITHUB_OUTPUT"
+        run: echo "version=$(jq .version VERSION.json)" >> "$GITHUB_OUTPUT"
 
       - name: Build with Gradle
         run: ./gradlew build -Dversion=${{ steps.version.outputs.version }} -Drevision=${{ inputs.revision }}

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -158,7 +158,7 @@ jobs:
 
       - name: Get version
         id: version
-        run: echo "version=$(<VERSION)" >> "$GITHUB_OUTPUT"
+        run: echo "version=$(bash build-scripts/product_version.sh)" >> "$GITHUB_OUTPUT"
 
       - name: Build with Gradle
         working-directory: ./plugins/${{ matrix.plugins }}
@@ -201,7 +201,7 @@ jobs:
 
       - name: Get version
         id: version
-        run: echo "version=$(<VERSION)" >> "$GITHUB_OUTPUT"
+        run: echo "version=$(bash build-scripts/product_version.sh)" >> "$GITHUB_OUTPUT"
 
       - name: Build with Gradle
         run: ./gradlew build -Dversion=${{ steps.version.outputs.version }} -Drevision=${{ inputs.revision }}

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -158,7 +158,7 @@ jobs:
 
       - name: Get version
         id: version
-        run: echo "version=$(jq .version VERSION.json)" >> "$GITHUB_OUTPUT"
+        run: echo "version=$(jq .version -r VERSION.json)" >> "$GITHUB_OUTPUT"
 
       - name: Build with Gradle
         working-directory: ./plugins/${{ matrix.plugins }}
@@ -201,7 +201,7 @@ jobs:
 
       - name: Get version
         id: version
-        run: echo "version=$(jq .version VERSION.json)" >> "$GITHUB_OUTPUT"
+        run: echo "version=$(jq .version -r VERSION.json)" >> "$GITHUB_OUTPUT"
 
       - name: Build with Gradle
         run: ./gradlew build -Dversion=${{ steps.version.outputs.version }} -Drevision=${{ inputs.revision }}

